Sources/UtilityBeltNetworking/HTTPClient+Combine.swift

```diff
@@ -7,20 +7,39 @@
     @available(iOS 13, macOS 10.15, tvOS 13.0, watchOS 6, *)
     public extension HTTPClient {
         // MARK: Data Response
+        
+        /// A function that validates the output from a DataTaskPublisher.  Throwing an error indicates a failed validation.
+        typealias Validator = (URLSession.DataTaskPublisher.Output) throws -> Void
+        
+        private enum Validators {
+            /// Creates a validator that will ensure the response has a specific mime type
+            /// - Parameter mimeType: The mime type we want to validate the existance of.
+            /// - Returns: The validator that will do this check.
+            static func ensureMimeType(_ mimeType: MimeType) -> Validator {
+                return { element in
+                    let responseMimeType = element.response.mimeType
+                    if responseMimeType != mimeType.rawValue {
+                        throw UBNetworkError.invalidContentType(responseMimeType ?? "unknown")
+                    }
+                }
+            }
+        }
```

Comment:
I wouldn't mind having `Validators` in its own file! Especially given the other suggestion of reusing this across both clients.